tests/test_wsfev1.py

```diff
@@ -40,6 +41,35 @@
 pytestmark =[pytest.mark.vcr, pytest.mark.freeze_time('2021-07-01')]
 
 
+@pytest.fixture
+def mock_client():
```

Comment:
This fixture could go inside the test (if it not will be reused), or at least have a more descriptive name.
This would improve readability (and if possible keep the code near of where it is used, just a personal preference)

Reply:
Done 
I have made the changes as well